openlibrary/plugins/openlibrary/js/swagger-ui.js

```diff
@@ -0,0 +1,20 @@
+import SwaggerUIBundle from 'swagger-ui-dist/swagger-ui-bundle.js';
+import SwaggerUIStandalonePreset from 'swagger-ui-dist/swagger-ui-standalone-preset.js';
```

Comment:
Can you try  `import { SwaggerUIBundle, SwaggerUIStandalonePreset } from "swagger-ui-dist"`

Reply:
> Can you try `import { SwaggerUIBundle, SwaggerUIStandalonePreset } from "swagger-ui-dist"`

Yes, i can make these changes.